cmake/extensions.cmake

```diff
@@ -415,6 +415,47 @@ macro(zephyr_library_named name)
   target_link_libraries(${ZEPHYR_CURRENT_LIBRARY} PUBLIC ${IMAGE}zephyr_interface)
 endmacro()
 
+# Determines what the current directory's lib name would be and writes
+# it to the argument "lib_name" as if ZEPHYR_BASE had been equal to
+# ZEPHYR_CURRENT_MODULE_DIR.
+macro(zephyr_library_amend_module_lib_name lib_name)
+  message(FATAL_ERROR "Function only available for Zephyr modules.")
```

Comment:
Is this error missing an if-statement ?

Reply:
good spotted, thansk.
This was added at last minut, cause I wanted to safe guard accidential calls from Zephyr repo.
Guess it was done a little to fast.

Reply:
Fixed